### PR TITLE
LPS-195650 - Automation | Objects

### PR DESCRIPTION
**Pull request by thaiscabrals**
Ticket: [LPS-195650](https://liferay.atlassian.net/browse/LPS-195650)
Test: ObjectFolder#CanMoveObjectDefinitionToCurrentFolder

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFolder.testcase [hunk 1]
```diff
@@ -23,6 +23,56 @@ definition {
...
+				objectName = "CustomObject185681",
+				pluralLabelName = "Custom Objects 185681");
+		}
+
+		task ("When the user navigates to Folder A and moves the object definition to the current folder") {
+			ObjectAdmin.openObjectAdmin();
+
+			WaitForElementPresent(
+				folderLabel = "Folder A",
+				locator1 = "ObjectAdmin#SIDEBAR_ITEM");
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFolder.testcase:54]: @thaiscabrals did the test fail without the wait?

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Hey @thaiscabrals, can we rewrite and make sure we are following the steps in https://liferay.atlassian.net/browse/LPS-197996?focusedCommentId=2480396? There should be a "Move to Current Folder" option in the sidebar

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
ci:test:sf

**Review by timpak — APPROVED** [review]
LGTM!

**Comment by timpak** [comment]
@thaiscabrals can you update the commit message for your last commit? `add` should be capitalized since it's the first word in the commit message. Once you do that, you can forward this.

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
The test was sent to this [PR](https://github.com/liferay-bpm-qa/liferay-portal/pull/1221).

**Pull request closed without merging**